### ISSUE TITLE
local images are not rendered

### ISSUE THREAD

**Issue by tatoosh** [shd101wyy/markdown-preview-enhanced#476]
Local images in assets folder will not be shown in preview.

**Comment by 7Moses** [comment]
I am probably seeing something similar, e.g., ![](/Users/moses/Dropbox/.../___.jpg  "") does yield an image in the Preview; if I am missing something please let me know;  markdown-preview-enhanced is beautiful work so it has crossed my thoughts that there maybe something in the Documents concerning this,

**Comment by shd101wyy** [comment]
I assume you guys are using Windows?

**Comment by tatoosh** [comment]
@shd101wyy yes, Windows 10 x64 with Atom 1.18.0 ia32 and markdown-preview-enhanced 0.13.0.
Do oyu know the problem?

**Comment by tmori3y2** [comment]
Maybe, there is a relationship between # 348.

**Comment by 7Moses** [comment]
Still seeing the Issue. I am using Markdown Preview Enhanced 0.13.0, Atom 1.18.0, on Mac OS X Yosemite (10.10.5).

~~~~  
![Figure](/Users/___/Dropbox/GlobalInbox_Dropbox/___.jpg  "")   
~~~~  

Thanks

**Comment by shd101wyy** [comment]
@7Moses this is not supported.   
```markdown
![Figure](/Users/___/Dropbox/GlobalInbox_Dropbox/___.jpg  "")   
```

You can only write like this:  
```markdown
Related to current markdown file path
![](image.png)
![](./image.png)

Related to current project directory path
![](/image.png)
```

I might support below in the future:  
```markdown
Add `file://`
![Figure](file:///Users/___/Dropbox/GlobalInbox_Dropbox/___.jpg  "")   
```

**Comment by 7Moses** [comment]
Thank you!! I will read the support docs more closely next time,

**Comment by lalloni** [comment]
I'm having the same problem, on Linux, Ubuntu 18.04.

**Comment by DTayloe** [comment]
For anyone else experiencing this issue, you must "Open Folder" in VS Code so that you can see the markdown file and images/folder of images for this extension to have access.
